restricted/hasher/statetrie.go

```diff
@@ -214,7 +214,7 @@ func (st *StackTrie) TryUpdate(key, value []byte) error {
 
 func (st *StackTrie) Update(key, value []byte) {
 	if err := st.TryUpdate(key, value); err != nil {
-		fmt.Errorf("Unhandled trie error in StackTrie.Update", "err", err)
+		fmt.Println("Unhandled trie error in StackTrie.Update:", err)
```

Comment:
We should handle this with log.Error() rather than fmt.Println.

Log information goes to stderr, while fmt.Println goes to stdout. In general, information about how a process is running should be on stderr, while information that is the work product of a process running should go to stdout.